src/reference/00-Getting-Started/03-Directories.md

```diff
@@ -50,18 +46,31 @@ src/
 Other directories in `src/` will be ignored. Additionally, all hidden
 directories will be ignored.
 
+Source code can be placed in the project's base directory as
+`hello/app.scala`, which may be for small projects,
+though for normal projects people tend to keep the projects in
+the `src/main/` directory to keep things neat.
+The fact that you can place `*.scala` source code might seem like
```

Comment:
should be "The fact that you can place `*.scala` source code *in the projects the base directory*"?